contrib/ansible/README.md

```diff
@@ -0,0 +1,43 @@
+# Build a Kubernetes cluster using k3s via Ansible.
+
+## Kubeadm Ansible Playbook
```

Comment:
Kubeadm?

Reply:
Hummm *habits* .... Fix in [b97dc6b](https://github.com/rancher/k3s/pull/217/commits/b97dc6be4f95be098b24993081a8a0844dacec40)

Reply:
:+1: